client/src/components/dashboard/TemplateSelector.js

```diff
@@ -4,67 +4,40 @@
  */
 import React, { useState } from 'react';
 import {
-  Container, Button, Form, Col, Image,
+  Container, Button, Form, Col, Card,
 } from 'react-bootstrap';
-import { useDispatch } from 'react-redux';
+import { useDispatch, useSelector } from 'react-redux';
 import { PropTypes } from 'prop-types';
-import { updateTeamTheme } from '../../actions/team';
+import { updateTheme } from '../../actions/website';
 
 // Picture of each layout
-import singleColumnLayout from '../../images/theme1.png';
-import fShapeLayout from '../../images/f-shape-layout.png';
-import zigZagLayout from '../../images/zig-zag-layout.png';
+import layout1 from '../../images/layout_1.png';
+import layout2 from '../../images/layout_2.png';
+import layout3 from '../../images/layout_3.png';
 
 /**
  * Form for user input github credentials and select template.
  */
 const TemplateSelector = (props) => {
   const dispatch = useDispatch();
 
-  // Storing and passing Form Inputs, theme1 & layout1 as defualt
-  const [formInputs, setInputs] = useState({
-    layout: 1,
-    primaryColor: '#419aee',
-    secondaryColor: '#8da4d1',
-  });
-  const [theme, setTheme] = useState(1);
+  /* eslint-disable no-unused-vars */
+  const [darkMode, setDarkMode] = useState(useSelector((state) => state.website.darkTheme));
+  const [layout, setLayout] = useState(useSelector((state) => state.website.layout));
```

Comment:
After selecting the layout and update it, the layout chocie always reverted back to layout 1 after refreshing the page. I think you could use useEffect hook here to fix this problem

Reply:
Thanks for the suggestion!